install/kubernetes/helm/istio/charts/galley/values.yaml

```diff
@@ -29,3 +29,6 @@ tolerations: []
 # "security" and value "S1".
 podAntiAffinityLabelSelector: []
 podAntiAffinityTermLabelSelector: []
+
+# Enable reconcile validatingwebhookconfiguration
+enableReconcileWebhookConfiguration: true
```

Comment:
this is a user facing doc, can you explain what this means and why a user would set it?

Reply:
Done.